### DISCUSSION TITLE
Download link expired

### DISCUSSION THREAD

**Discussion by basedpainnn** [Pokemon-PythonRed/Pokemon-PythonRed#9] [General]
Hi!

When I click on the download link, it tells me that the download link has expired, with Correlation ID ce2cfd9f-0060-0000-c696-31627cb194be. Is there any way to remedy this? Thanks!

![Screenshot (758)](https://user-images.githubusercontent.com/93128221/139019362-1ce7a137-8007-49b9-a8d3-c66d359ddd01.png)

**Comment by riverpaper** [comment]
Thanks for letting me know! My organisation only lets me keep a link for a maximum of about 4 months, and I haven't worked on this project recently due to school. I'm contemplating using a different method of making the game, as just the first little bit has been a lot of effort. I've made a lot of plans for this game, but I don't have the time to program it. But it's nice to see someone showing interest! Just know that there's not very much so far, and what's already done will likely be changed if I do come back to it. Thanks again, I'll update the link soon!

EDIT: for the moment, the game will no longer use Releases. Also, the game *is* being rewritten.
